packages/features/bookings/Booker/components/hooks/useBookings.ts

```diff
@@ -25,6 +25,7 @@ export interface IUseBookings {
   hashedLink?: string | null;
   bookingForm: UseBookingFormReturnType["bookingForm"];
   metadata: Record<string, string>;
+  teamMemberEmail?: string;
```

Comment:
This is the email of the team member that is the owner of the contact in Salesforce. Pass it to `handleNewBooking` via this variable.